File: find-missing/findMissing.js
const findMissing = (array1, array2) => {
    const sort1 = [...array1].sort((a, b) => { return a - b;} );
    const sort2 = [...array2].sort((a, b) => { return a - b;} );
    
    for(let i = 0; i < sort1.length; i++){
        if(sort1[i] !== sort2[i]){
            return sort1[i];
        };
    };
};

module.exports = { findMissing };


//sort both arrays -
//start loop - when [i] on each array differs, return arr1[i]

File: python-venv/find_missing/kata_function/find_missing.py
def find_missing(array1, array2):
    sort1 = array1
    sort1.sort()
    sort2 = array2
    sort2.sort()

    for i in range(len(sort1)):
        if i == len(sort1) - 1:
            return sort1[len(sort1) - 1]
        if sort1[i]!= sort2[i]:
            return sort1[i]




# def find_missing(arr1, arr2):
    # return sum(arr1) - sum(arr2)